cypress/integration/workspace.test.ts

```diff
@@ -5,7 +5,7 @@ context("Test the overall app", () => {
 
   describe("Desktop functionalities", () => {
     it("renders with text", () => {
-      cy.get(".app").should("have.text", "Hello World");
+      cy.get(".app").should("exist");
```

Comment:
Testing for existence works; but, I'd prolly test for the outer-most enclosing div's name -- since if anybody changes it from "app", the top react render would be busted. But a totally minor nit-pick.

Reply:
I'm anticipating a lot more Cypress test changes will come in future, and we're likely to change the hierarchy in future, but having a working mobx component on the landing page is a reasonable way to test that pieces are working for now.